src/popup/components/ProtocolIcon.vue

```diff
@@ -35,10 +37,21 @@ export default defineComponent({
       default: 'rg',
       validator: (val: AllowedProtocolIconSize) => SIZES.includes(val),
     },
+    /**
+     * Not all icons have secondary version
+     * Secondary version is used for example when displaying token icon
+     */
+    iconVariant: {
+      type: String as PropType<AllowedProtocolIconVariations>,
+      default: 'default',
+    },
```

Comment:
Maybe instead use bool prop so the default variant would be the "coin" logo and if the bool prop would be passed the protocol "logo" would be used.